diff --git a/plexapi/audio.py b/plexapi/audio.py
index 370fe0d..e2ce5b9 100644
--- a/plexapi/audio.py
+++ b/plexapi/audio.py
@@ -225,9 +225,11 @@ class Artist(
 
     def albums(self, **kwargs):
         """ Returns a list of :class:`~plexapi.audio.Album` objects by the artist. """
+        filters = kwargs.pop('filters', {})
+        filters['artist.id'] = self.ratingKey
         return self.section().search(
             libtype='album',
-            filters={**kwargs.pop('filters', {}), 'artist.id': self.ratingKey},
+            filters=filters,
             **kwargs
         )
 
