diff --git a/plexapi/mixins.py b/plexapi/mixins.py
index ea6a075..40d2324 100644
--- a/plexapi/mixins.py
+++ b/plexapi/mixins.py
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 from collections import deque
 from datetime import datetime
-from typing import Tuple
+from typing import Deque, Set, Tuple, Union
 from urllib.parse import parse_qsl, quote, quote_plus, unquote, urlencode, urlsplit
 
 from plexapi import media, settings, utils
 from plexapi.exceptions import BadRequest, NotFound
 from plexapi.utils import deprecated, openOrRead
 
+from backports.datetime_timestamp import timestamp
+
 
 class AdvancedSettingsMixin:
     """ Mixin for Plex objects that can have advanced settings. """
@@ -66,10 +68,10 @@ class SmartFilterMixin:
     """ Mixin for Plex objects that can have smart filters. """
 
     def _parseFilterGroups(
-        self, feed: "deque[Tuple[str, str]]", returnOn: "set[str]|None" = None
+        self, feed: Deque[Tuple[str, str]], returnOn: Union[Set[str], None] = None
     ) -> dict:
         """ Parse filter groups from input lines between push and pop. """
-        currentFiltersStack: list[dict] = []
+        currentFiltersStack = []
         operatorForStack = None
         if returnOn is None:
             returnOn = set("pop")
@@ -563,9 +565,9 @@ class AddedAtMixin(EditFieldMixin):
                 locked (bool): True (default) to lock the field, False to unlock the field.
         """
         if isinstance(addedAt, str):
-            addedAt = int(round(datetime.strptime(addedAt, '%Y-%m-%d').timestamp()))
+            addedAt = int(round(timestamp(datetime.strptime(addedAt, '%Y-%m-%d'))))
         elif isinstance(addedAt, datetime):
-            addedAt = int(round(addedAt.timestamp()))
+            addedAt = int(round(timestamp(addedAt)))
         return self.editField('addedAt', addedAt, locked=locked)
 
 
