diff --git a/plexapi/playlist.py b/plexapi/playlist.py
index c435613..9143a0b 100644
--- a/plexapi/playlist.py
+++ b/plexapi/playlist.py
@@ -385,7 +385,7 @@ class Playlist(
         try:
             return server.playlists(sectionId=section.key, guid__endswith=m3ufilepath)[0].edit(title=title).reload()
         except IndexError:
-            raise BadRequest('Failed to create playlist from m3u file.') from None
+            raise BadRequest('Failed to create playlist from m3u file.')
 
     @classmethod
     def create(cls, server, title, section=None, items=None, smart=False, limit=None,
