diff --git a/plexapi/server.py b/plexapi/server.py
index 69d5f89..20cab42 100644
--- a/plexapi/server.py
+++ b/plexapi/server.py
@@ -287,7 +287,7 @@ class PlexServer(PlexObject):
         try:
             return next(account for account in self.systemAccounts() if account.id == accountID)
         except StopIteration:
-            raise NotFound(f'Unknown account with accountID={accountID}') from None
+            raise NotFound(f'Unknown account with accountID={accountID}')
 
     def systemDevices(self):
         """ Returns a list of :class:`~plexapi.server.SystemDevice` objects this server contains. """
@@ -305,7 +305,7 @@ class PlexServer(PlexObject):
         try:
             return next(device for device in self.systemDevices() if device.id == deviceID)
         except StopIteration:
-            raise NotFound(f'Unknown device with deviceID={deviceID}') from None
+            raise NotFound(f'Unknown device with deviceID={deviceID}')
 
     def myPlexAccount(self):
         """ Returns a :class:`~plexapi.myplex.MyPlexAccount` object using the same
@@ -701,7 +701,7 @@ class PlexServer(PlexObject):
         try:
             return self.playlists(title=title, title__iexact=title)[0]
         except IndexError:
-            raise NotFound(f'Unable to find playlist with title "{title}".') from None
+            raise NotFound(f'Unable to find playlist with title "{title}".')
 
     def optimizedItems(self, removeAll=None):
         """ Returns list of all :class:`~plexapi.media.Optimized` objects connected to server. """
