diff --git a/plexapi/myplex.py b/plexapi/myplex.py
index c90b5d3..cfed7c0 100644
--- a/plexapi/myplex.py
+++ b/plexapi/myplex.py
@@ -225,7 +225,7 @@ class MyPlexAccount(PlexObject):
         method = method or self._session.get
         timeout = timeout or TIMEOUT
         log.debug('%s %s %s', method.__name__.upper(), url, kwargs.get('json', ''))
-        headers = self._headers(**headers or {})
+        headers = self._headers(**(headers or {}))
         response = method(url, headers=headers, timeout=timeout, **kwargs)
         if response.status_code not in (200, 201, 204):  # pragma: no cover
             codename = codes.get(response.status_code)[0]
