diff --git a/plexapi/server.py b/plexapi/server.py
index 69d5f89..19783f9 100644
--- a/plexapi/server.py
+++ b/plexapi/server.py
@@ -597,7 +597,7 @@ class PlexServer(PlexObject):
                     print("Available butler tasks:", availableTasks)
 
         """
-        validTasks = [task.name for task in self.butlerTasks()]
+        validTasks = [_task.name for _task in self.butlerTasks()]
         if task not in validTasks:
             raise BadRequest(
                 f'Invalid butler task: {task}. Available tasks are: {validTasks}'
