diff --git a/plexapi/client.py b/plexapi/client.py
index 2b4283c..bbc9bbc 100644
--- a/plexapi/client.py
+++ b/plexapi/client.py
@@ -181,7 +181,7 @@ class PlexClient(PlexObject):
         method = method or self._session.get
         timeout = timeout or TIMEOUT
         log.debug('%s %s', method.__name__.upper(), url)
-        headers = self._headers(**headers or {})
+        headers = self._headers(**(headers or {}))
         response = method(url, headers=headers, timeout=timeout, **kwargs)
         if response.status_code not in (200, 201, 204):
             codename = codes.get(response.status_code)[0]
