diff --git a/plexapi/utils.py b/plexapi/utils.py
index d1882fb..b5ce505 100644
--- a/plexapi/utils.py
+++ b/plexapi/utils.py
@@ -7,7 +7,6 @@ import os
 import re
 import string
 import time
-import unicodedata
 import warnings
 import zipfile
 from collections import deque
@@ -26,6 +25,8 @@ try:
 except ImportError:
     tqdm = None
 
+from plexapi.backports import makedirs, unicodedata_normalize
+
 log = logging.getLogger('plexapi')
 
 # Search Types - Plex uses these to filter specific media types when searching.
@@ -357,7 +358,7 @@ def toList(value, itemcast=None, delim=','):
 
 def cleanFilename(filename, replace='_'):
     whitelist = f"-_.()[] {string.ascii_letters}{string.digits}"
-    cleaned_filename = unicodedata.normalize('NFKD', filename).encode('ASCII', 'ignore').decode()
+    cleaned_filename = unicodedata_normalize('NFKD', filename).encode('ASCII', 'ignore').decode()
     cleaned_filename = ''.join(c if c in whitelist else replace for c in cleaned_filename)
     return cleaned_filename
 
@@ -431,7 +432,7 @@ def download(url, token, filename=None, savepath=None, session=None, chunksize=4
 
     # make sure the savepath directory exists
     savepath = savepath or os.getcwd()
-    os.makedirs(savepath, exist_ok=True)
+    makedirs(savepath, exist_ok=True)
 
     # try getting filename from header if not specified in arguments (used for logs, db)
     if not filename and response.headers.get('Content-Disposition'):
