diff --git a/plexapi/mixins.py b/plexapi/mixins.py
index f0c21cf..800536c 100644
--- a/plexapi/mixins.py
+++ b/plexapi/mixins.py
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from backports.datetime_timestamp import timestamp
 from datetime import datetime
 from urllib.parse import parse_qsl, quote, quote_plus, unquote, urlencode, urlsplit
 
@@ -529,9 +530,9 @@ class AddedAtMixin(EditFieldMixin):
                 locked (bool): True (default) to lock the field, False to unlock the field.
         """
         if isinstance(addedAt, str):
-            addedAt = int(round(datetime.strptime(addedAt, '%Y-%m-%d').timestamp()))
+            addedAt = int(round(timestamp(datetime.strptime(addedAt, '%Y-%m-%d'))))
         elif isinstance(addedAt, datetime):
-            addedAt = int(round(addedAt.timestamp()))
+            addedAt = int(round(timestamp(addedAt)))
         return self.editField('addedAt', addedAt, locked=locked)
 
 
