diff --git a/plexapi/library.py b/plexapi/library.py
index cbca424..f3f3427 100644
--- a/plexapi/library.py
+++ b/plexapi/library.py
@@ -450,7 +450,18 @@ class LibrarySection(PlexObject):
         # based on the item type being batch multi-edited
         value = super().__getattribute__(attr)
         if attr.startswith('_'): return value
-        if callable(value) and 'Mixin' in value.__qualname__:
+        import sys
+        _qualname = ''
+        if sys.version_info < (3, 3):
+            from qualname import qualname
+            try:
+                _qualname = qualname(value)
+            except Exception:
+                pass
+        else:
+            if hasattr(value, '__qualname__'):
+                _qualname = value.__qualname__
+        if callable(value) and 'Mixin' in _qualname:
             if not isinstance(self._edits, dict):
                 raise AttributeError("Must enable batchMultiEdit() to use this method")
             elif not hasattr(self._edits['items'][0], attr):
