diff --git a/setup.py b/setup.py
index 7e87e12..c81820c 100644
--- a/setup.py
+++ b/setup.py
@@ -23,23 +23,24 @@ with open('requirements.txt') as handle:
     requirements = [str(req) for req in parse_requirements(handle)]
 
 setup(
-    name='PlexAPI',
+    name='PlexAPI-backport',
     version=version['__version__'],
     description='Python bindings for the Plex API.',
-    author='Michael Shepanski',
-    author_email='michael.shepanski@gmail.com',
-    url='https://github.com/pkkid/python-plexapi',
+    author='LizardByte',
+    author_email='LizardByte@github.com',
+    url='https://github.com/LizardByte/python-plexapi-backport',
     packages=['plexapi'],
     install_requires=requirements,
     extras_require={
-        'alert': ["websocket-client>=1.3.3"],
+        'alert': ["websocket-client>=0.59.0"],
     },
-    python_requires='>=3.8',
+    python_requires='>=2.7',
     long_description=readme,
     long_description_content_type='text/x-rst',
     keywords=['plex', 'api'],
     classifiers=[
         'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'License :: OSI Approved :: BSD License',
     ]
