diff --git a/setup.py b/setup.py
index fb50e2e..56fb381 100644
--- a/setup.py
+++ b/setup.py
@@ -23,19 +23,20 @@ with open('requirements.txt') as handle:
     requirements = [str(req) for req in parse_requirements(handle)]
 
 setup(
-    name='PlexAPI',
+    name='PlexAPI-backport',
     version=version['__version__'],
     description='Python bindings for the Plex API.',
-    author='Michael Shepanski',
-    author_email='michael.shepanski@gmail.com',
-    url='https://github.com/pkkid/python-plexapi',
+    author='LizardByte',
+    author_email='LizardByte@ithub.com',
+    url='https://github.com/LizardByte/python-plexapi-backport',
     packages=['plexapi'],
     install_requires=requirements,
-    python_requires='>=3.8',
+    python_requires='>=2.7',
     long_description=readme,
     keywords=['plex', 'api'],
     classifiers=[
         'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'License :: OSI Approved :: BSD License',
     ]
