diff --git a/tests/conftest.py b/tests/conftest.py
index 386d081..9d871d2 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -141,7 +141,8 @@ def account_once(account):
 def account_plexpass(account):
     if not account.subscriptionActive:
         pytest.skip(
-            "PlexPass subscription is not active, unable to test sync-stuff, be careful!"
+            "PlexPass subscription is not active, unable to test dashboard, movie extras, movie editions, "
+            "or sync-stuff, be careful!"
         )
     return account
 
diff --git a/tests/test_myplex.py b/tests/test_myplex.py
index 28e862a..5ebd2dd 100644
--- a/tests/test_myplex.py
+++ b/tests/test_myplex.py
@@ -77,6 +77,11 @@ def test_myplex_users(account):
     print(f"Found user: {user}")
     assert user, f"Could not find user {users[0].title}"
 
+    try:
+        users[0].servers[0]
+    except IndexError:
+        return pytest.skip(f"{users[0].title} shared user does not have access to any servers")
+
     assert (
         len(users[0].servers[0].sections()) > 0
     ), "Couldn't info about the shared libraries"
diff --git a/tests/test_server.py b/tests/test_server.py
index e1bcf9f..6768466 100644
--- a/tests/test_server.py
+++ b/tests/test_server.py
@@ -439,7 +439,7 @@ def test_server_system_devices(plex):
     
 
 @pytest.mark.authenticated
-def test_server_dashboard_bandwidth(plex):
+def test_server_dashboard_bandwidth(account_plexpass, plex):
     bandwidthData = plex.bandwidth()
     assert len(bandwidthData)
     bandwidth = bandwidthData[0]
@@ -456,7 +456,7 @@ def test_server_dashboard_bandwidth(plex):
 
 
 @pytest.mark.authenticated
-def test_server_dashboard_bandwidth_filters(plex):
+def test_server_dashboard_bandwidth_filters(account_plexpass, plex):
     at = datetime(2021, 1, 1)
     filters = {
         'at>': at,
diff --git a/tests/test_video.py b/tests/test_video.py
index fd9f962..3537aee 100644
--- a/tests/test_video.py
+++ b/tests/test_video.py
@@ -586,7 +586,7 @@ def test_video_Movie_editions(movie):
 
 
 @pytest.mark.authenticated
-def test_video_Movie_extras(movies):
+def test_video_Movie_extras(account_plexpass, movies):
     movie = movies.get("Sita Sings The Blues")
     extras = movie.extras()
     assert extras
@@ -677,7 +677,7 @@ def test_video_Movie_mixins_fields_edition(movie):
 
 
 @pytest.mark.authenticated
-def test_video_Movie_mixins_fields_edition_authenticated(movie):
+def test_video_Movie_mixins_fields_edition_authenticated(account_plexpass, movie):
     test_mixins.edit_edition_title(movie)
 
 
