diff --git a/tools/plex-bootstraptest.py b/tools/plex-bootstraptest.py
index 79287db..81ee811 100755
--- a/tools/plex-bootstraptest.py
+++ b/tools/plex-bootstraptest.py
@@ -15,9 +15,7 @@ import os
 import shutil
 import socket
 import time
-from glob import glob
-from os import makedirs
-from shutil import copyfile, which
+from shutil import copyfile
 from subprocess import call
 from uuid import uuid4
 
@@ -28,6 +26,12 @@ from plexapi.server import PlexServer
 from plexapi.utils import SEARCHTYPES
 from tqdm import tqdm
 
+try:
+    from shutil import which
+except ImportError:
+    from backports.shutil_which import which
+from plexapi.backports import glob, makedirs
+
 DOCKER_CMD = [
     "docker",
     "run",
